test/ExpressiveInstallerTest/RoutersTest.php

```diff
@@ -93,9 +106,27 @@ public function testRouter(
         );
 
         // Test home page
-        $setupRoutes = (strpos($copyFilesKey, 'minimal') !== 0);
+        $setupRoutes = strpos($copyFilesKey, 'minimal') !== 0;
         $response = $this->getAppResponse('/', $setupRoutes);
         $this->assertEquals($expectedResponseStatusCode, $response->getStatusCode());
+
+        /** @var Application $app */
+        $app = $container->get(Application::class);
+        $this->assertCount(count($expectedRoutes), $app->getRoutes());
+        foreach ($app->getRoutes() as $route) {
+            foreach ($expectedRoutes as $expectedRoute) {
+                if ($expectedRoute['name'] === $route->getName()) {
+                    $this->assertEquals($expectedRoute['path'], $route->getPath());
+                    // todo: not sure how we can test middleware?
+                    // $this->assertEquals($expectedRoute['middleware'], $route->getMiddleware());
```

Comment:
The purpose of this test is to ensure that the routes are registered, and that the application can route to them correctly, which is why we only look at status codes. For the non-minimal setups, a 404 would indicate a problem with the configuration; for the minimal setup, a 200 would indicate a problem with the configuration.

I do see your point about not splitting up `getAppResponse()`, and would be fine with keeping it as-is, and simply calling it from within the `foreach` loop.